x-pack/plugins/cases/public/containers/api.test.tsx

```diff
@@ -268,14 +267,115 @@ describe('Cases API', () => {
         queryParams: DEFAULT_QUERY_PARAMS,
         signal: abortCtrl.signal,
       });
+
       expect(fetchMock).toHaveBeenCalledWith(`${CASES_URL}/_find`, {
         method: 'GET',
         query: {
           ...DEFAULT_QUERY_PARAMS,
           searchFields: DEFAULT_FILTER_OPTIONS.searchFields,
+        },
+        signal: abortCtrl.signal,
+      });
+    });
+
+    test('should apply the severity field correctly (with status value)', async () => {
+      await getCases({
+        filterOptions: {
+          ...DEFAULT_FILTER_OPTIONS,
+          status: CaseStatuses.open,
+        },
+        queryParams: DEFAULT_QUERY_PARAMS,
+        signal: abortCtrl.signal,
+      });
+
+      expect(fetchMock).toHaveBeenCalledWith(`${CASES_URL}/_find`, {
+        method: 'GET',
+        query: {
+          ...DEFAULT_QUERY_PARAMS,
+          searchFields: DEFAULT_FILTER_OPTIONS.searchFields,
+          status: CaseStatuses.open,
+        },
+        signal: abortCtrl.signal,
+      });
+    });
+
+    test('should not send the severity field with "all" status value', async () => {
+      await getCases({
+        filterOptions: {
+          ...DEFAULT_FILTER_OPTIONS,
+          status: 'all',
+        },
+        queryParams: DEFAULT_QUERY_PARAMS,
+        signal: abortCtrl.signal,
+      });
+
+      expect(fetchMock).toHaveBeenCalledWith(`${CASES_URL}/_find`, {
+        method: 'GET',
+        query: {
+          ...DEFAULT_QUERY_PARAMS,
+          searchFields: DEFAULT_FILTER_OPTIONS.searchFields,
+        },
+        signal: abortCtrl.signal,
+      });
+    });
+
+    test('should not send the assignees field if it an empty array', async () => {
```

Comment:
Just to clarify, the `fetchMock` will be called with an empty `assignees` array right? If so then maybe we change the test text to say `should send an empty assignees field when it is an empty array for the filters options` or something like that.

Reply:
The `fetchMock` should not be called with an empty `assignees` array. If you pass an empty array the URL will contain unnecessary `&`. For example `api/cases/_find?&&sortOrder='desc'`. Do you want me to rephrase the text to make it clearer?

Reply:
ah I misread the `expect`. Sounds good!